android/java/org/chromium/chrome/browser/playlist/sources.gni

```diff
@@ -5,6 +5,96 @@
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/model/LastPlayedPositionModel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/model/MoveOrCopyModel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/model/PlaylistItemModel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/model/PlaylistItemOptionModel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/model/PlaylistModel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/model/PlaylistOnboardingModel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/model/PlaylistOptionsModel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/model/RenamePlaylistModel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/model/SnackBarActionModel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/playback_service/VideoPlaybackService.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/slidingpanel/BottomPanelLayout.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/slidingpanel/ScrollableViewHelper.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/slidingpanel/ViewDragHelper.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/util/ConstantUtils.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/util/HLSParsingUtil.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/util/MediaItemUtil.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/util/MediaUtils.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/util/MenuUtils.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/util/PlaylistItemGestureHelper.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/util/PlaylistPreferenceUtils.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/util/PlaylistUtils.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/util/PlaylistViewUtils.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/view/MovableImageButton.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/view/PlaylistOnboardingPanel.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/view/PlaylistToolbar.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/view/bottomsheet/MoveOrCopyToPlaylistBottomSheet.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/view/bottomsheet/PlaylistItemOptionsBottomSheet.kt",
+  "../../brave/android/java/org/chromium/chrome/browser/playlist/kotlin/view/bottomsheet/PlaylistOptionsBottomSheet.kt",
+]
+
+playlist_java_sources = [
```

Comment:
same as above, these should be added to their own BUILD.gn (one for each directory)

Reply:
Fixed in https://github.com/brave/brave-core/pull/26628/commits/e257593acc6cbfaa8c06228c44d674fc0dc3b97b